openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala

```diff
@@ -306,14 +339,34 @@ object OpenApi {
         mutable.LinkedHashMap(
           "openapi" -> ujson.Str(openApiVersion),
           "info" -> infoJson(openApi.info),
-          "paths" -> pathsJson(openApi.paths)
+          "paths" -> pathsJson(openApi.paths),
         )
+      if (openApi.tags.nonEmpty) {
+       fields += "tags" -> ujson.Arr(openApi.tags.map(tagJson): _*)
+      }
       if (openApi.components.schemas.nonEmpty || openApi.components.securitySchemes.nonEmpty) {
         fields += "components" -> componentsJson(openApi.components)
       }
       new ujson.Obj(fields)
     }
 
+  private def extractTags(paths: Map[String, PathItem]): List[Tag] = {
```

Comment:
Should we return a `Set[Tag]`?

Reply:
Yes we could do that, I made it a list since that is what ujson expects, that conversion can also happen later.